tests/cupy_tests/core_tests/test_elementwise.py

```diff
@@ -92,14 +92,14 @@ class TestElementwiseType(unittest.TestCase):
     @testing.numpy_cupy_array_equal()
     def test_large_int_upper_1(self, xp, dtype):
         a = xp.array([0], dtype=xp.int8)
-        b = xp.iinfo(dtype).max
+        b = xp.asarray(xp.iinfo(dtype).max)
```

Comment:
Sorry, I overlooked this during my previous review.
These tests are for the type promotion of ndarray and Python primitive scalar. could you please fix it as follows to check if numpy and cupy throw the same exception?

```py
    @testing.for_int_dtypes(no_bool=True)
    @testing.numpy_cupy_array_equal(accept_error=OverflowError)
    def test_large_int_upper_1(self, xp, dtype):
        a = xp.array([0], dtype=xp.int8)
        b = xp.iinfo(dtype).max
        return a + b
```

Reply:
OK, done.

Reply:
Could you fix also test_large_int_upper_{2,3,4}?

Reply:
OK, fixed test_large_int_upper_2 and test_large_int_lower_{1,2}. 
_{3,4} variants are identical here and in main and all pass, at least locally.